.husky/pre-push

```diff
@@ -11,4 +11,4 @@ if [ $EXIT_CODE == 1 ]; then
 fi
 
 npm run typecheck
-npm run depcruise
+npm run depcruise
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline